Tools/WinMLRunner/OutputHelper.h

```diff
@@ -354,7 +407,9 @@ class OutputHelper
                      << "Wall-clock Load (ms)" << ","
                      << "Wall-clock Bind (ms)" << ","
                      << "Wall-clock Evaluate (ms)" << ","
-                     << "Wall-clock total time (ms)" << std::endl;
+                     << "Wall-clock total time (ms)" << ","
+                     << "PerIterationFile" << ","
```

Comment:
For users that primarily use -perf flag to capture performance and do not care about the per iterations and the results file, we don't want this in their file because it will mostly be blank.